arrow-meta/arrow-meta-test-models/build.gradle

```diff
@@ -10,8 +10,8 @@ apply from: "$DOC_CONF"
 
 dependencies {
     compile "org.jetbrains.kotlin:kotlin-stdlib-jdk8:$KOTLIN_VERSION"
-    compileOnly "com.google.auto.service:auto-service:$GOOGLE_AUTO_SERVICE_VERSION"
-    kapt "com.google.auto.service:auto-service:$GOOGLE_AUTO_SERVICE_VERSION"
+    compileOnly "com.google.auto.service:auto-service:1.0-rc7"
+    kapt "com.google.auto.service:auto-service:1.0-rc7"
```

Comment:
Thanks for this awesome PR @hadilq :raised_hands: I was thinking about this kind of changes and I think the mistake was to locate all the versions in a centralized place without filtering them.

I mean, in this case, `GOOGLE_AUTO_SERVICE_VERSION` (and `GOOGLE_COMPILE_TESTING_VERSION` as well) is just being used in this repository so I think it doesn't make sense to be placed in `arrow` repository.

So please, feel free to locate those versions in this repository and remove them from `arrow` repository. 

I think `arrow` repository should contain versions that are relevant and can impact to all the libraries. What do you think about it?

Thanks again @hadilq !!

Reply:
Thank you @rachelcarmena for your this great suggestion. I'm totally agree and you can find the changes in b8d1703 and https://github.com/arrow-kt/arrow/pull/2158

Reply:
Awesome, thanks @hadilq for your contribution!!